src/components/groupBrowsing/GroupBrowsingCard/GroupBrowsingCard.tsx

```diff
@@ -0,0 +1,255 @@
+import React, { FC } from 'react';
+import { styled } from 'stitches.config';
+import UserIcon from '@assets/svg/user.svg';
+import CalendarIcon from '@assets/svg/calendar.svg';
+import Avatar from '@components/avatar/Avatar';
+import { Flex } from '@components/util/layout/Flex';
+import { fonts } from '@sopt-makers/fonts';
+import { GroupBrowsingCardDetail, parsePartValueToLabel } from '@api/meeting';
+import dayjs from 'dayjs';
+import { ACTION_STATUS, EActionStatus } from '@constants/option';
+import Link from 'next/link';
+
+const GroupBrowsingCard: FC<GroupBrowsingCardDetail> = ({
+  id,
+  userId,
+  title,
+  category,
+  imageURL,
+  startDate,
+  endDate,
+  capacity,
+  mStartDate,
+  mEndDate,
+  recentActivityDate,
+  targetActiveGeneration,
+  joinableParts,
+  applicantCount,
+  approvedUserCount,
+  user,
+  status,
+}) => {
+  const isAllParts = joinableParts?.length === 6 || joinableParts === null;
+  const isGroupActive = new Date(mStartDate) <= new Date() && new Date() <= new Date(mEndDate);
```

Comment:
`import dayjs from 'dayjs';` 한 김에
```suggestion
  const isGroupActive = dayjs().isBetween(dayjs(mStartDate), dayjs(mEndDate));
```
[isBetween](https://day.js.org/docs/en/plugin/is-between)을 활용하는 건 어떨까?

Reply:
헉 너무 좋다!!! 🤩